packages/schema/src/types/config.ts

```diff
@@ -45,4 +45,9 @@ export interface ViteConfig extends ViteUserConfig {
    * @see https://github.com/vitejs/vite/tree/main/packages/plugin-vue
    */
   vue?: VuePluginOptions
+  /**
+   * Bundler for dev time server-side rendering.
+   * @default 'vite-node'
+   */
+  serverBundler?: 'vite-node' | 'legacy'
```

Comment:
Maybe:


```suggestion
  devBundler?: 'vite-node' | 'legacy'
```

To make it clear we are not talking about anything runtime in built app.